### DISCUSSION TITLE
Welcome to Taproot Assets Discussions!

### DISCUSSION THREAD

**Discussion by Roasbeef** [lightninglabs/taproot-assets#265] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BitTogether** [comment]
Bittogether here, excited to build for the taproot asset community.  Does anybody have a guide for creating or buying taproot assets using this?
